resources/js/contest-voting/entry.coffee

```diff
@@ -61,11 +61,19 @@ export class Entry extends React.Component
       if @props.contest.show_votes
         div className: 'contest-voting-list__title contest-voting-list__title--show-votes',
           div className: 'contest-voting-list__votes-bar', style: { width: "#{relativeVotePercentage}%" }
-          div className: 'u-relative u-ellipsis-overflow', entry_title
+          div className: 'u-relative u-ellipsis-overflow',
+            a
+              className: 'contest-voting-list__title--link',
+              href: route('beatmapsets.show', beatmapset: @props.entry.preview)
```

Comment:
you pointed this out on discord already but just reminding, yeah this is a special case for `submitted_beatmaps` contests and `@props.entry.preview` would otherwise be a url on its own. deciding the URL should share logic with the existing icon links on line 50 & 57